package.json

```diff
@@ -11,9 +11,9 @@
     "build-client-prod": "HOST=127.0.0.1 PORT=3333 PROTOCOL=http NODE_ENV=production webpack --watch --progress",
     "build-server-prod": "babel --watch src/server -d dist/server",
     "watch-client": "webpack --watch --progress",
-    "watch-server": "babel --watch src/server -d dist/server",
+    "watch-server": "babel --watch src/server -d dev/server",
```

Comment:
What's up with this? Seems like it could be a pretty disruptive change for people (like me) running everything on dist/prod